packages/components/src/RatioBar/RatioBar.scss

```diff
@@ -13,21 +9,21 @@ $ratio-bar-hover-height: 8px;
 .tc-ratio-bar {
 	padding: $padding-small 0;
 	display: flex;
-	height: $ratio-bar-hover-height;
-	width: $ratio-bar-width;
+	height: tokens.$coral-spacing-xs;
```

Comment:
Please use `sizing` tokens for sizes 🙏

Reply:
If they're not in your bundle, you're using an outdated version of the design tokens :)

Reply:
Oh yes, you're right, I got confused!
But `tokens.$coral-spacing-xs` = 0.8rem and the smaller size we have is `tokens.$coral-sizing-minimal` = 1.2rem 🤔 
What do you suggest to do then? keep it as it is for now? revert it to a custom value?

Reply:
If I remember correctly, @AnLefevre did specify this would be an exception to the token scale. Nobody likes them, but they may occur... When they do, we'd advise for custom values clearly labeled as such (`$custom-ratio-bar-height: 0.8rem`) so we can track all `custom` stuff down the line.

Reply:
Ok thanks for the explanation, I'll go for that then 👍